script/update_beats.sh

```diff
@@ -48,5 +50,9 @@ cp $GIT_CLONE/.go-version .go-version
 # Get system test dependencies
 cp -r $GIT_CLONE/testing testing
 
+# Generate .gitignore file
+echo "libbeat/_meta/fields.generated.yml" > .gitignore
+echo "libbeat/fields.yml" >> .gitignore
```

Comment:
Would it maybe be better to remove those files again instead of adding them to `.gitignore`?

Reply:
Removing does not work as these files are generated during `make update` and I want to prevent someone else checking them in.